docs/tutorials/advanced_cogshop_run.ipynb

```diff
@@ -878,19 +900,19 @@
       "\n",
       "INFORMATION: 1047\n",
       "\n",
-      "Current time: Tue Oct  8 07:27:04 2024\n",
+      "Current time: Mon Oct 21 08:26:14 2024\n",
       "\n",
       "\n",
       "\n",
-      "DIAGNOSIS WARNING: 3250\n",
+      "WARNING: 1239\n",
       "\n",
-      "Reservoir Dalbysvatn: Only max tactical cost or limit is given.\n",
+      "Object generator named Sola_G1 needs to define attribute(s): p_nom\n",
```

Comment:
Kanskje vi burde ha lagt til `p_nom`...
Tror 85.7 og 86.6 er fornuftige verdier for hhv. G1 og G2